src/app/ApplicationTemplate.Tests/Presentation/ChuckNorrisSearchPageViewModelShould.cs

```diff
@@ -12,65 +12,44 @@
 
 namespace ApplicationTemplate.Tests
 {
-	public class ChuckNorrisSearchPageViewModelTests : NavigationTestsBase
+	public class ChuckNorrisSearchPageViewModelShould : NavigationTestsBase
 	{
-		[Fact]
-		public async Task It_Should_EmptyCriterion_NoResults()
+		private void MockingGetFavorites(Mock<IChuckNorrisService> mock)
 		{
-			// Arrange
-			var viewModel = new ChuckNorrisSearchPageViewModel();
-			viewModel.SearchTerm = string.Empty;
-
-			// Act
-			var quotes = await viewModel.Quotes.Load(DefaultCancellationToken);
-
-			// Assert
-			using (new AssertionScope())
-			{
-				quotes.Should().NotBeNull();
-				quotes.Should().BeEmpty();
-			}
+			mock
+				.Setup(m => m.GetFavorites(It.IsAny<CancellationToken>()))
+				.ReturnsAsync(new SourceList<ChuckNorrisQuote>().AsObservableList());
 		}
 
-		[Fact]
-		public async Task It_Should_NotEmptyCriterion_Results()
+		// Basic class configuration
```

Comment:
The comment says Basic configuration but the method name says Special configuration. Which is it?